src/NDSCart_SRAMManager.cpp

```diff
@@ -121,27 +120,52 @@ namespace NDSCart_SRAMManager
                 continue;
             }
 
-            FlushSecondaryBufferToFile();
+            FlushSecondaryBuffer();
         }
     }
     
-    void FlushSecondaryBufferToFile()
+    void FlushSecondaryBuffer(u8* dst, s32 dstLength)
```

Comment:
how/when are these parameters filled in?

Reply:
When called externally, e.g. from BizHawk. I have a local branch with all of that integration which I will make a PR for at some point.